scripts/common.py

```diff
@@ -468,28 +468,32 @@ def convert_to_posixpath(file_path):
     return file_path.replace('\\', '/')
 
 
-def inplace_replace_file(filename, regex_pattern, replace):
-    """Replace the file content inplace with regex pattern. The pattern is used
+def inplace_replace_file(filename, regex_pattern, replacement_string):
+    """Replace the file content inp-lace with regex pattern. The pattern is used
     to replace the file's content line by line.
 
     Note:
-        This function was not designed to achive a comprehensive task. Instead,
+        This function was not designed to achieve a comprehensive task. Instead,
```

Comment:
This note is still too vague. Please make it clearer. You can say "This function should only be used with files that are processed line by line." and that suffices (right?)

Otherwise developers will not know whether their task is "trivial" or "comprehensive". It's really important to be more specific here.

Reply:
Done